var/spack/repos/builtin/packages/codecov/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import platform
+
+from spack.package import *
+
+
+class Codecov(Package):
+    """Codecov uploads coverage reports to Codecov for processing."""
+
+    homepage = "https://codecov.io"
+
+    versions = {
```

Comment:
You should either change the name or move it outside the class. Spack uses `self.versions` to store all versions.